heron/schedulers/tests/java/com/twitter/heron/scheduler/marathon/MarathonControllerTest.java

```diff
@@ -34,14 +34,16 @@
 @PrepareForTest(NetworkUtils.class)
 public class MarathonControllerTest {
   private static final String MARATHON_URI = "http://marathon.uri:8080";
+  private static final String MARATHON_AUTH_TOKEN = null;
```

Comment:
Yes, I can try to add a test here where i'm looking for the specific response code